### DISCUSSION TITLE
A presentation mode of Notion page

### DISCUSSION THREAD

**Discussion by tianfanghan** [marp-team/marp#280] [Ideas]
There is just an idea. Nowadays note apps just like Notion is hot. I wonder is it possible to make a system like wunderpresentation.com to render the Notion shared pages directly into presentation mode? wunderpresentation.com is doing good, but not that customized.

Thank you agian, for your hard works.

**Comment by yhatt** [comment]
Our goal is for making the open-sourced presentation ecosystem that is targeted for the Markdown format. It means we are focusing for making the framework and base system for building slides. So every developers can try to create the tool for converting Notion pages to Marp slides.

On the other hand, we would have to waste a lot of times to make a system because we have to deal with a lot of differences between the block-based Notion note and the Marp Markdown format. Currently we have no times to maintain Web service together with our ecosystem. We believe the open source community to extend Marp's power. :)
